internal/checker/checker.go

```diff
@@ -22,29 +22,33 @@ import (
 // repository (e.g. missing blobs), and needs a valid Repository to work on.
 type Checker struct {
 	packs    restic.IDSet
-	blobs    restic.IDSet
 	blobRefs struct {
 		sync.Mutex
-		M map[restic.ID]uint
+		// see flags below
+		M map[restic.BlobHandle]blobBits
 	}
-	indexes map[restic.ID]*repository.Index
 
 	masterIndex *repository.MasterIndex
 
 	repo restic.Repository
 }
 
+type blobBits uint8
+
+const (
+	existsFlag blobBits = 1 << iota
+	referencedFlag
```

Comment:
I would call the type blobStatus, or just status, and the values statusExists and statusReferenced. The stdlib nearly always has its enum values start with the type name.

Reply:
Thanks for the suggestion, I've change the type name to `blobStatus` and change the value names accordingly.